tests/test_threadpoolctl.py

```diff
@@ -361,3 +362,29 @@ def test_blis_threading_layer():
 
     actual_layer = blis_info.modules[0].threading_layer
     assert actual_layer == expected_layer
+
+
+@pytest.mark.skipif(not cython_extensions_compiled,
+                    reason='Requires cython extensions to be compiled')
+def test_libomp_libiomp_warning(recwarn):
+    # Trigger the import of a potentially clang-compiled extension:
+    from ._openmp_test_helper import check_nested_openmp_loops  # noqa
+
+    # Trigger the import of numpy to potentially import Intel OpenMP via MKL
+    import numpy.linalg  # noqa
```

Comment:
This is not necessary since we already import numpy for the whole module

Reply:
explicit is better than implicit :)